src/questionGenerator.js

```diff
@@ -21,6 +20,7 @@ const questions = [
 
 const used = [
   'Window seat or aisle seat?',
+  'Which movie do you think has the best soundtrack?',
```

Comment:
Yep, this one too! 
'What is your go-to breakfast? Is it sweet or savory?',

Reply:
Consider it moved

Reply:
Addressed code review change request(s) through: [2923772](https://github.com/mwall01/weekly-wall/pull/3/commits/29237729a25ba6904d22d822c1ea39b2b3aa929c)

Reply:
**Resolving Conversation**